examples/bgm111/ble-scan/app/start.sh

```diff
@@ -0,0 +1,11 @@
+echo "Loading ftdi_sio module..."
```

Comment:
we should avoid suggesting using this vs the fin block for flashing the co-processor

Reply:
@curcuz I'll create an issue for this